.ci/azure-pipelines/build-windows.yml

```diff
@@ -27,16 +29,28 @@ jobs:
       - script: |
           vcpkg.exe install eigen3 flann gtest qhull --triplet %PLATFORM%-windows && vcpkg.exe list
         displayName: 'Install Dependencies'
+      - script: |
+          curl -L https://github.com/frerich/clcache/releases/download/v4.2.0/clcache.4.2.0.nupkg --output clcache.4.2.0.nupkg
```

Comment:
Maybe extract version to a `CLCACHE_VERSION` variable so that future updates are quicker.

Reply:
Future updates, as in? Is the intent to cache the download?

Reply:
@SunBlack mentioned that there will maybe be a new version of `clcache` soon with improved speed. I just thought that bumping the version in a single variable is easier/cleaner for us than three times in a string.